extutils/imgproc/apng2gif.py

```diff
@@ -64,30 +66,38 @@ def _get_file_name(file_path: str) -> str:
     return os.path.splitext(os.path.basename(file_path))[0]
 
 
-def _extract_frames(result: ConvertResult, file_path: str):
+def _extract_frames(result: ConvertResult, file_path: str) -> Optional[List[Tuple[bytes, Fraction]]]:
```

Comment:
Codacy found an issue: [Either all return statements in a function should return an expression, or none of them should.](https://app.codacy.com/manual/RaenonX/Jelly-Bot/pullRequest?prid=6015490)